integration-tests/wallet-manager-that-calls-failwith-contract.spec.ts

```diff
@@ -6,13 +6,13 @@ import { MANAGER_LAMBDA } from "@taquito/taquito";
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
 
-  describe(`Test contract that calls 2nd contract that FAILs: ${rpc}`, () => {
+  describe(`Test contract origination of a contract that calls 2nd contract that FAILs through wallet api: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('Catches a Failwith from a contract made with wallet api called via a manager contract',  async (done) => {
+    test('Verify that wallet.originate for a contract with code FAILWITH that links to another contract will fail', async (done: () => void) => {
```

Comment:
I feel like the description is not totally accurate. It is not wallet.originate that fails.
We successfully originate a contract having a FAILWITH instruction in its code. (contract)
We successfully originate a manager contract. (managerContract))
We call the do method of the managerContract to transfer 1 token to the contract, and expect the operation to throw.

I suggest: `Verify that transferring token from the manager contract to a contract having a FAILWITH instruction will fail.`

Reply:
changed